package.json

```diff
@@ -120,6 +120,8 @@
     "typescript": "4.6.2"
   },
   "dependencies": {
+    "@xrengine/engine": "^0.4.13",
```

Comment:
please remove the `@xrengine/engine` dep from here, and move `detect-gpu` to /packages/engine/package.json